src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java

```diff
@@ -102,19 +102,13 @@ public static boolean isInterpreterRequiredHearingOptions(HearingOptions hearing
     public static List<CaseCategory> buildCaseCategories(SscsCaseData caseData,
                                                          ReferenceDataServiceHolder referenceDataServiceHolder) throws ListingException {
         // TODO Adjournment - Check this is the correct logic for Adjournment
-        SessionCategoryMap sessionCaseCode = HearingsMapping.getSessionCaseCode(caseData, referenceDataServiceHolder);
+        SessionCategoryMap sessionCategoryMap = HearingsMapping.getSessionCaseCode(caseData, referenceDataServiceHolder);
```

Comment:
Minor - it could be worth renaming the getSessionCaseCode() to reflect we're returning a map (so getSessionCaseCodeMap()) to be a bit more specific? Not sure if this method was originally not returning a map but might make things a tiny bit clearer.

Reply:
Changed to getSessionCaseCodeMap